[bug] test_plot_conformal_prediction fails with auto-regression enabled on plot_latest_forecast

Here is the code snippet from `test_plot_conformal_prediction` in the test_plotting.py file:

```
    # With auto-regression enabled
    # TO-DO: Fix Assertion error n_train >= 1
    m = NeuralProphet(
        n_forecasts=7,
        n_lags=14,
        quantiles=[0.05, 0.95],
        epochs=EPOCHS,
        batch_size=BATCH_SIZE,
        learning_rate=LR,
    )
    train_df, test_df = m.split_df(df, freq="D", valid_p=0.2)
    train_df, cal_df = m.split_df(train_df, freq="D", valid_p=0.15)
    metrics_df = m.fit(train_df, freq="D")
    alpha = 0.1
    for method in ["naive", "cqr"]:  # Naive and CQR SCP methods
        future = m.make_future_dataframe(df, periods=m.n_forecasts, n_historic_predictions=10)
        forecast = m.conformal_predict(future, calibration_df=cal_df, alpha=alpha, method=method)
        m.highlight_nth_step_ahead_of_each_forecast(m.n_forecasts)
        fig0 = m.plot(forecast)
        fig1 = m.plot_latest_forecast(forecast, include_previous_forecasts=10, plotting_backend="matplotlib")
        fig2 = m.plot_latest_forecast(forecast, include_previous_forecasts=10, plot_history_data=True, plotting_backend="matplotlib")
        fig3 = m.plot_latest_forecast(forecast, include_previous_forecasts=10, plot_history_data=False, plotting_backend="matplotlib")
        fig4 = m.plot_components(forecast, plotting_backend="matplotlib")
        fig5 = m.plot_parameters(plotting_backend="matplotlib")
        if PLOT:
            fig0.show()
            fig1.show()
            fig2.show()
            fig3.show()
            fig4.show()
            fig5.show()
```
Running the test gets this gets the error:

```
FAILED tests/test_plotting.py::test_plot_conformal_prediction[matplotlib] - IndexError: index 5 is out of bounds for axis 1 with size 5
FAILED tests/test_plotting.py::test_plot_conformal_prediction[plotly] - IndexError: index 5 is out of bounds for axis 1 with size 5
```

This is due to the `m.plot_latest_forecast()` method, where somewhere in the utils.py or df_utils.py file, the conformal predict column (e.g., `yhat1 5.0% + qhat1`) is not correctly mapped.

--- a/neuralprophet/utils.py
+++ b/neuralprophet/utils.py
@@ -586,7 +586,7 @@ def fcst_df_to_latest_forecast(fcst, quantiles, n_last=1):
     df = pd.concat((fcst[cols],), axis=1)
     df.reset_index(drop=True, inplace=True)
 
-    yhat_col_names = [col_name for col_name in fcst.columns if "yhat" in col_name and "%" not in col_name]
+    yhat_col_names = [col_name for col_name in fcst.columns if "yhat" in col_name and "%" not in col_name and "qhat" not in col_name]
     yhat_col_names_quants = [col_name for col_name in fcst.columns if "yhat" in col_name and "%" in col_name]
     n_forecast_steps = len(yhat_col_names)
     yhats = pd.concat((fcst[yhat_col_names],), axis=1)

--- a/neuralprophet/utils.py
+++ b/neuralprophet/utils.py
@@ -586,7 +586,9 @@ def fcst_df_to_latest_forecast(fcst, quantiles, n_last=1):
     df = pd.concat((fcst[cols],), axis=1)
     df.reset_index(drop=True, inplace=True)
 
-    yhat_col_names = [col_name for col_name in fcst.columns if "yhat" in col_name and "%" not in col_name and "qhat" not in col_name]
+    yhat_col_names = [
+        col_name for col_name in fcst.columns if "yhat" in col_name and "%" not in col_name and "qhat" not in col_name
+    ]
     yhat_col_names_quants = [col_name for col_name in fcst.columns if "yhat" in col_name and "%" in col_name]
     n_forecast_steps = len(yhat_col_names)
     yhats = pd.concat((fcst[yhat_col_names],), axis=1)

